packages/flutter/lib/src/widgets/_web_selectable_region_context_menu_io.dart

```diff
@@ -0,0 +1,41 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+// Fake interface for the `WebSelectableRegionContextMenu` widget that requires
+// html library. See `_web_selectable_region_context_menu_web.dart` for web
+// implementation.
+
+import 'framework.dart';
+import 'selection_container.dart';
+
+/// Function signature for `ui.platformViewRegistry.registerViewFactory`.
+@visibleForTesting
+typedef RegisterViewFactory = void Function(String, Object Function(int viewId), {bool isVisible});
+
+/// A mock widget for non-web platform.
+///
+/// See `_web_selectable_region_context_menu_web.dart` for the widget used in web
+/// platform.
+class WebSelectableRegionContextMenu extends StatelessWidget {
```

Comment:
Suggestion: rename this to `PlatformSelectableRegionContextMenu`. I think the purpose of this class is less about the "web" and more about communicating the selectable region to the platform. It's just that it happens that the web implementation uses it for something, and others are (as of today) noop.